packages/runtime/lib/schema.js

```diff
@@ -191,7 +191,7 @@ const platformaticRuntimeSchema = {
       type: 'array',
       items: {
         type: 'object',
-        required: ['id', 'path', 'config'],
+        required: ['id', 'path'],
```

Comment:
why?

Reply:
Because we now allow generic application that might not have any configuration file at all.